app/src/utils/RedirectionUtils.js

```diff
@@ -46,12 +46,13 @@ export const redirectToCreateRuleEditor = (navigate, rule) => {
 };
 
 /* FEATURE - RULES - Edit a Rule */
-export const redirectToRuleEditor = (navigate, ruleId, source, newTab = false) => {
+export const redirectToRuleEditor = (navigate, ruleId, source, newTab = false, popCurrentRouteFromHistory = false) => {
```

Comment:
suggestions: (since the function now has more parameters to control the redirect)
1. rename argument to `replaceCurrentRouteInHistory`
2. add a doc string on this function explaining this param, so that it's use is clearly accessible via tha linter highlight